src/client/src/apps/MainRoute/widgets/spotTile/components/PriceControls/PriceControls.tsx

```diff
@@ -20,6 +20,7 @@ interface Props {
   disabled: boolean
   rfqState: RfqState
   isTradeExecutionInFlight: boolean
+  analyticsView?: boolean
```

Comment:
I know we are not quite consistent there, but I find calling boolean props `is...` or `has...` (in this case `isAnalyticsView`) a bit more obvious when reading the code.